src/App.tsx

```diff
@@ -1,156 +1,78 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
+import { TodoHeader } from './Components/TodoHeader';
+import { TodoMain } from './Components/TodoMain';
+import { TodoFooter } from './Components/TodoFooter';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
 
-const USER_ID = 0;
+const USER_ID = 11143;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isError, setIsError] = useState('');
+  const [filterType, setFilterType] = useState('all');
+  const [query, setQuery] = useState('');
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(setTodos)
+      .catch(setIsError);
+  }, []);
+
+  const preparedTodos = [...todos]
```

Comment:
use `useMemo` hook

Reply:
[not fixed](https://github.com/mate-academy/react_todo-app-loading-todos/pull/697/files#r1277466998)